docs/content/latest/yugabyte-cloud/cloud-network/vpc-peers/_index.md

```diff
@@ -0,0 +1,85 @@
+---
+title: VPC Peering
+headerTitle: VPC Peering
+linkTitle: VPC Peering
+description: Configure VPC peering.
+image: /images/section_icons/secure/tls-encryption/connect-to-cluster.png
+headcontent: Set up VPC peering for your Yugabyte Cloud clusters.
+menu:
+  latest:
+    identifier: vpc-peers
+    parent: cloud-network
+    weight: 400
+---
+
+Virtual Private Cloud (VPC) peering allows applications running on instances on the same cloud provider as your Yugabyte Cloud clusters to communicate with those clusters without traversing the public internet; all traffic stays within the cloud provider's network. Yugabyte Cloud supports AWC and GCP for self-managed peering. For Azure, contact Yugabyte Cloud support.
```

Comment:
Need to add benefits of using VPCs.  VPC improves network security, reduces network latency and cost